src/Components/Results/Results.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react';
+import './Results.css';
+import { PostData } from '../../Types/ResultsTypes';
+import { postNewForm, PostInfo } from '../../apiCalls';
+import Loader from '../Loader/Loader';
+
+interface ResultsProps {
+  currentResult: PostData | null
+  updateCurrentResult: (result: PostData) => void
+  formData: PostInfo | null
+}
+
+const Results = ({currentResult, formData, updateCurrentResult}: ResultsProps) => {
+  const [loading, setLoading] = useState(false)
+  if (!currentResult) {
+    return (<div>no results</div>)
```

Comment:
I think we should come back to this and style this some more- maybe add a link to the form to prompt the user to go fill out it out or something?

Reply:
Yes! I think Fernando just had this as a placeholder for now